### PR TITLE
update daemon service url

### PR DESCRIPTION
**Pull request by gindis**
fix daemon startup prompt and update document.

### CODE REVIEW

File: packages/cli/src/bin/pipcook-daemon.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ async function start(): Promise<void> {
     if (message.event === 'ready') {
       daemon.disconnect();
       daemon.unref();
-      logger.success(`Pipcook is on http://localhost:${message.data.listen}`);
+      logger.success('Pipcook service started, please visit https://pipboard.vercel.app');
```
Comment (FeelyChau) [inline packages/cli/src/bin/pipcook-daemon.ts:39]: I think the daemon url should not be removed, if we access daemon from remote by `@pipcook/pipcook-cli`, like:
```sh
$ pipcook job list -h 192.168.1.33 -p 1234
```
The `port` will miss.

**Review by yorkie — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ $ pipcook run https://raw.githubusercontent.com/alibaba/pipcook/master/example/p
 If you are wondering what you can do in [Pipcook][] and where you can check your training logs and models, you could start from [Pipboard](https://alibaba.github.io/pipcook/#/GLOSSORY?id=pipboard):
 
 ```
-open http://localhost:6927/index.html
+open https://pipboard.vercel.app/index.html
```
Comment (yorkie) [inline README.md:128]:
```suggestion
open https://pipboard.vercel.app/
```

**Review by FeelyChau — CHANGES_REQUESTED** [review]
@gindis Thanks for your contribution!

File: packages/cli/src/bin/pipcook-daemon.ts (hunk 1 above)
Comment (gindis) [reply packages/cli/src/bin/pipcook-daemon.ts:39]: Now visit [http://localhost:6927/](http://localhost:6927/) and 404 will appear. suggest port can be viewed on pipboard web.

File: README.md (hunk 2 above)
Comment (gindis) [reply README.md]: fixed

**Pull request closed without merging**